test/suite/test_hs02.py

```diff
@@ -42,8 +42,10 @@ class test_hs02(wttest.WiredTigerTestCase):
     session_config = 'isolation=snapshot'
 
     key_format_values = [
-        ('column', dict(key_format='r')),
-        ('string', dict(key_format='S'))
+        ('string', dict(key_format='S')),
+        # Remove for column store until rollback to stable is implemented for column store
```

Comment:
Styling/Formatting - I think these sets of comments need to be a completed sentence (ended with a period '.') ?